### ISSUE TITLE
using --allow-downgrade with Params seems not to work.

### ISSUE THREAD

**Issue by alkampfergit** [PowerShellOrg/cChoco#12]
With this configuration      

  cChocoPackageInstaller libreOffice
        {
            Name = "libreoffice"
            Version = "4.4.3"
            Params = "--allow-downgrade"
            DependsOn = "[cChocoInstaller]installChoco"
        }

I still got the error as --allow-downgrade was not specified.

Am I doing something wrong?

Thanks.

VERBOSE: [ADLDSDOMAIN]:                            [[cChocoPackageInstaller]libreOffice] Package output Installing the
following packages: libreoffice By installing you accept licenses for the packages. A newer version of libreoffice
(v5.0.2) is already installed.  **Use --allow-downgrade to attempt to install older versions**, or use side by side to
allow multiple versions.  Chocolatey installed 0/1 package(s). 1 package(s) failed.  See the log for details

**Comment by ferventcoder** [comment]
Params are the package params passed to choco, that get passed on to the package itself. It appears you need additional arguments passed to choco itself, which is a new field that I don't believe exists here yet.

See https://github.com/PowerShellOrg/cChoco/blob/master/DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1#L153

Side note: https://github.com/PowerShellOrg/cChoco/blob/master/DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1#L153 and surrounding code look like they could use some refactoring.

**Comment by alkampfergit** [comment]
Thanks for the clarification, I think that should not be a bit problem to fork and modify the package to add such a functionality.

**Comment by javydekoning** [comment]
This has been fixed in #53 

``` powershell
      cChocoPackageInstaller installSkypeWithChocoParams
      {
        Name                 = 'skype'
        chocoParams          = '--allowdowngrade'
        Ensure               = 'Present'
      }
```
